pytest.ini

```diff
@@ -14,17 +14,21 @@ filterwarnings=
 	# Ensure ResourceWarnings are emitted
 	default::ResourceWarning
 
+	# Workarounds for pypa/setuptools#3810
+	# Can't use EncodingWarning as it doesn't exist on Python 3.9
+	default:'encoding' argument not specified
+	# subprocess.check_output still warns with EncodingWarning even with encoding set
+	ignore:'encoding' argument not specified::setuptools.tests.environment
+
 	# realpython/pytest-mypy#152
 	ignore:'encoding' argument not specified::pytest_mypy
 
-	# python/cpython#100750
-	ignore:'encoding' argument not specified::platform
-
-	# pypa/build#615
-	ignore:'encoding' argument not specified::build.env
+	# pytest-dev/pytest # TODO: Raise issue upstream 
+	ignore:'encoding' argument not specified::_pytest
 
-	# dateutil/dateutil#1284
-	ignore:datetime.datetime.utcfromtimestamp:DeprecationWarning:dateutil.tz.tz
+	# pypa/distutils # TODO: Raise issue upstream 
```

Comment:
See pypa/distutils#150 (pending merge into Setuptools).

Reply:
I'm not sure how that PR relates to the following EncodingWarnings:

```
/home/runner/work/setuptools/setuptools/setuptools/_distutils/dist.py:398: EncodingWarning: 'encoding' argument not specified
  parser.read(filename)
    
/home/runner/work/setuptools/setuptools/setuptools/_distutils/text_file.py:118: EncodingWarning: 'encoding' argument not specified
  self.file = open(self.filename, errors=self.errors)
```

Maybe you meant https://github.com/pypa/distutils/pull/236 or https://github.com/pypa/distutils/issues/232 ?